Jenkinsfile

```diff
@@ -0,0 +1,132 @@
+pipeline {
+  agent any
+  options {
+    timeout(time: 1, unit: 'HOURS')
+  }
+  environment {
+    IMAGE_BASE = "build.datapunt.amsterdam.nl:5000/amaps/app"
+    IMAGE_BUILD = "${IMAGE_BASE}:${BUILD_NUMBER}"
+    IMAGE_ACCEPTANCE = "${IMAGE_BASE}:acceptance"
+    IMAGE_PRODUCTION = "${IMAGE_BASE}:production"
+    IMAGE_LATEST = "${IMAGE_BASE}:latest"
+  }
+  stages {
+    stage('Clean') {
+      // TODO remove this stage when jenkins jobs run in isolation
+      steps {
+        sh "docker ps"
+        sh "docker-compose down -v || true"
+        sh "docker network prune -f"
+        sh "docker-compose -p ${env.BRANCH_NAME} up --build build-nlmaps"
+      }
+    }
+    stage('Test & Bakkie') {
+      // failFast true // fail if one of the parallel stages fail
+      parallel {
+        stage('Linting') {
+          steps {
+            sh "docker network prune -f"
+            sh "docker-compose -p ${env.BRANCH_NAME} up --build --exit-code-from lint lint"
+          }
+        }
+        stage('Testing') {
+          steps {
+            sh "docker network prune -f"
+            sh "docker-compose -p ${env.BRANCH_NAME} up --build --exit-code-from test test"
+          }
+        }
+      }
+      post {
+        always {
+          sh "docker-compose -p ${env.BRANCH_NAME} down -v || true"
+        }
+      }
+    }
+    stage('Build A (Master)') {
```

Comment:
The part above looks fine to me, but I'm not sure if there's anything to be built here that would make sense to deploy? If I follow correctly, this will build the Dockerfile from this repo, which might not contain the nlmaps code. Also, while the test index.html could be served with a webserver, there is currently no default command that would do that.

Reply:
Yeah. I'll remove this for now. In a next step it would be good to have something to deploy as some kind of test/demo page.

Reply:
Yes, and I just thought of something too: this repo build also outputs the customized nlmaps library with Amsterdam's config. That library code needs to be deployed somewhere too: how would you guys want to do that? Push it to github, or to npm? The code now gets stored in a shared docker volume.

Reply:
Indeed. For now: deploy it together with the demo page. Later we can push it back to github and maybe also publish to npm.